src/test/java/com/epam/tc/hw7/JsonParserFile.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.tc.hw7;
+
+import com.epam.tc.hw7.entities.MetalColorEntity;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class JsonParserFile {
+
+    public LinkedHashMap fileToMap() {
```

Comment:
```suggestion
    public LinkedHashMap<KeyType, ValueType> fileToMap() {
```

Reply:
changed public LinkedHashMap fileToMap() to LinkedHashMap<String, MetalColorEntity> fileToMap()